app/views/home/index.html.erb

```diff
@@ -10,6 +10,7 @@
 <%= govuk_start_button(text: "Create a form", href: new_form_path) %>
 
 <h2 class="govuk-heading-m">Your forms</h2>
+
```

Comment:
Doesn't look like this actually includes the summary list change - think maybe something has gone wrong with a history rewrite?

Reply:
🤦  Yeah, I had commited the reverted changes to take screenshots. Should now see the changes